src/Makefile

```diff
@@ -34,13 +40,14 @@ DEFINES+=-DREMUSER=\"${OSSEC_USER_REM}\"
 DEFINES+=-DGROUPGLOBAL=\"${OSSEC_GROUP}\"
 DEFINES+=-DMAILUSER=\"${OSSEC_USER_MAIL}\"
 
-LDFLAGS+=-lm
+OSSEC_LDFLAGS=${LDFLAGS} -lm
```

Comment:
so you can call make for e.g.:
`make TARGET=server CC=my_gcc CFLAGS=my_flags LDFLAGS=my_flags2`
cause i had the experience that predefined variables can not be modified in a makefile